docs/management/managing-fields.asciidoc

```diff
@@ -72,16 +72,23 @@ WARNING: Computing data on the fly with scripted fields can be very resource int
 Kibana's performance. Keep in mind that there's no built-in validation of a scripted field. If your scripts are
 buggy, you'll get exceptions whenever you try to view the dynamically generated data.
 
-Scripted fields use the Lucene expression syntax. For more information,
-see {es-ref}modules-scripting-expression.html[
-Lucene Expressions Scripts].
+When you go to define a scripted field in Kibana, you'll be given a choice of scripting language, allowing you to pick 
```

Comment:
I'd put the emphasis on the current defaults/recommended choices. "When you define a scripted field in Kibana, you have a choice of scripting languages. Starting with 5.0, the default options are Lucene expressions and Painless. While you can use other scripting languages if you enable dynamic scripting for them in Elasticsearch, this is not recommended because they cannot be sufficiently sandboxed."